src/middlewares/getUserFromResourceServer.js

```diff
@@ -0,0 +1,16 @@
+
+const getUser = (accessToken, refreshToken, profile, done) => {
+  console.log(`user : ${JSON.stringify(profile)}`);
```

Comment:
Do we need this console statement?

Reply:
Thanks @MuhanguziDavid , 
we definitely do not need this! 
I have deleted the line and moved the whole function into [passport config file](https://github.com/andela/knights-bn-backend/pull/27/files#diff-081b3f12542f954a81ba72aa794ddf46)